nolus/assetlist.json

```diff
@@ -23,6 +23,286 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nolus/images/nolus.png"
       },
       "coingecko_id": "nolus"
+    },
+    {
+      "description": "axlUSDC transferred from the Osmosis mainnet that is supported on the Nolus chain",
+      "denom_units": [
+        {
+          "denom": "ibc/7FBDBEEEBA9C50C4BCDF7BF438EAB99E64360833D240B32655C96E319559E911",
+          "exponent": 0,
+          "aliases": [
+            "uusdc"
+          ]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/7FBDBEEEBA9C50C4BCDF7BF438EAB99E64360833D240B32655C96E319559E911",
+      "name": "axlUSDC",
+      "display": "usdc",
+      "symbol": "axlUSDC",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858",
+            "channel_id": "channel-783"
+          },
+          "chain": {
+            "channel_id": "channel-0",
+            "path": "transfer/channel-0/transfer/channel-208/uusdc"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nolus/images/usdc.svg",
```

Comment:
Can I encourage to instead point the url to the respective origin chains that contain the images? e.g., https://raw.githubusercontent.com/cosmos/chain-registry/master/**_non-cosmos/ethereum**/images/usdc.svg, and not to upload duplicate images?

This way we don't need to store extra versions of images. I'd recommend only uploading more images if Nolus wants to override to original images.

In fact, image URLs here I'd consider optional, since the trace will point to the origin asset, which has the logo_URIs property.

Reply:
Done